### DISCUSSION TITLE
New TV

### DISCUSSION THREAD

**Discussion by janply** [NAIST-SE/AttractingContributors#617] [General]
I'm installed a new TV in my second floor bedroom, and also bought a separate firestick for it. I tried to adjust it, however, the TV doesn't see the remote. What should I do?

**Comment by XelaAl** [comment]
It seems you need to pair the remote. Did you surf Google?

**Comment by McJul** [comment]
Have you managed to solve the problem? I was searching to fix the issues with my streaming device due to it's slow performance, and came across with the [https://www.firesticktricks.com/pair-firestick-remote.html](https://www.firesticktricks.com/pair-firestick-remote.html) article. Hope it'll be useful for you.
